first_year_scraper/main.py

```diff
@@ -0,0 +1,22 @@
+import csv
+import excel_parser
+import excel_formatter
+import restructure
+
+filename = input("Enter the pdf file name: ")
+target_filename = "workbook.xlsx"
```

Comment:
So, usually it is a good idea to have constant filenames at the top of the file right after imports. It helps the developers to know what are the constants that are being used.

Reply:
Yea I'll keep that in mind from now on!